lib/BloqadeWaveforms/test/waveform.jl

```diff
@@ -192,7 +200,9 @@ end
     pwl = piecewise_linear(;clocks=Float64[0,1],values=Float64[1,4])
     @test pwl[0.0..0.5] ≈ piecewise_linear(clocks=Float64[0,0.5],values=Float64[1,2.5])
     @test pwl[0.5..1.0] ≈ piecewise_linear(clocks=Float64[0,0.5],values=Float64[2.5,4])
+    @test pwl[0.0..0.0] ≈ piecewise_linear(clocks=Float64[0,0.0],values=Float64[1,1])
```

Comment:
Both parts of the approximate equality (`pwl[0.0..0.0]` and `piecewise_linear(...)`) cause `Interpolations.jl` to issue the following warning: 
```
┌ Warning: Duplicated knots were deduplicated. Use Interpolations.deduplicate_knots!(knots) explicitly to avoid this warning.
│   k1 =
│    2-element Vector{Float64}:
│     0.0
│     0.0
└ @ Interpolations ~/.julia/packages/Interpolations/y4lLj/src/gridded/gridded.jl:77
```